qcodes/instrument/group_parameter.py

```diff
@@ -212,19 +211,29 @@ def parser(ret_str: str) -> Dict[str, Any]:
 
         return parser
 
-    @deprecate("set is confusingly implemented and should not be part "
-               "of the public api")
-    def set(self, set_parameter: GroupParameter,
-            raw_value: ParamRawDataType) -> None:
+    def set_parameters(self,
+                       parameters_dict: Dict[str, ParamRawDataType]) -> None:
         """
-        Sets the value of the given parameter within a group to the given
-        value by calling the ``set_cmd``.
+        Sets the value of one or more parameters within a group to the given
+        values by calling the ``set_cmd`` while updating rest.
 
         Args:
-            set_parameter: The parameter within the group to set.
-            raw_value: The new raw_value for this parameter.
+             parameters_dict: The dictionary of one or more parameters within
```

Comment:
I think there is an extra space on this line. The lines under `Args:` should be indented with exactly 4 extra spaces

Reply:
ok :) made the required changes just now.